pages/events/index.vue

```diff
@@ -83,9 +83,38 @@
           </div>
         </template>
         <template slot="extra-info">
-          <div class="event-page__start-an-event">
+          <div class="event-page__section">
+            <h3>Follow our event calendar</h3>
+            <p class="event-page__section__description">
+              <!-- eslint-disable-next-line vue/singleline-html-element-content-newline -->
+              Stay up to date with all of our scheduled events by following our calendar. You can view the calendar by visiting <AppLink v-bind="qiskitCalendarLink">{{ qiskitCalendarLink.label }}</AppLink>, or subscribe to it by adding to the calendar app of your choice.
+            </p>
+            <div class="event-page__tabs">
+              <cv-tabs aria-label="navigation tab label">
```

Comment:
The `aria-label` attribute should label the tabs. It could be something like "Calendar apps".

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tab_role#example

Reply:
Updated in https://github.com/Qiskit/qiskit.org/pull/2955/commits/95fe85daeb5f39f4ece6f4de2aef27381c5d1cd5